.dockerignore

```diff
@@ -1,7 +1,6 @@
 .env
 .env.j2
 .env.template
-.env.test
```

Comment:
This is a reasonable change but I don't think it should be necessary - you also perform an alternative by passing the test environment variables through at runtime with `--env-file .env.test`. Is there any reason to do both? Is there any disadvantage in doing both?